pyvista/core/utilities/features.py

```diff
@@ -99,6 +99,114 @@ def voxelize(mesh, density=None, check_surface=True):
     return vox
 
 
+def voxelize_volume(mesh, density=None, check_surface=True):
+    """Voxelize mesh to create a RectilinearGrid voxel volume.
+
+    Creates a voxel volume that encloses the input mesh and discretizes the cells
+    within the volume that intersect and are contained within the input mesh.
```

Comment:
My understanding of this function is that it selects all cells if any one point is selected inside the mesh.  If so, the below would work, or consider wording it like the previous sentence.

```suggestion
    within the volume that intersect or are contained within the input mesh.
```